oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -62,8 +59,17 @@ public static void main(String[] args) {
             RpService rpService = injector.getInstance(RpService.class);
             rpService.load();
 
+            //check multiple options
+            if(Cli.checkForMultipleOptions(cmd)) {
```

Comment:
`Cli` can be skipped. We in the same class. Add space after `if` (as it is within rest of the code.)

Reply:
Cli has been removed and space added after if.